ops/phase1/run.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -x
+set -e
+
+# if [ -z ${$1+x} ]; then echo "var is unset"; exit 1; else echo "var is set to '$1'"; fi
+# wget https://releases.hashicorp.com/terraform/0.13.5/terraform_0.13.5_linux_amd64.zip
+# unzip terraform_0.13.5_linux_amd64.zip
+# mv -f terraform /usr/local/bin
+[ ! -f service_principal.json ] && ./create-service-principle.sh
+sp=$(cat service_principal.json)
+
+echo "exporting terraform creds"
+export ARM_CLIENT_ID=$(echo $sp | jq -r '.appId')
+export ARM_CLIENT_SECRET=$(echo $sp | jq -r '.password')
+export ARM_SUBSCRIPTION_ID=$(az account show --query id --output tsv)
+export ARM_TENANT_ID=$(echo $sp | jq -r '.tenant')
+export TF_VAR_operator_subscription_id=$ARM_SUBSCRIPTION_ID
+export TF_VAR_operator_client_id=$ARM_CLIENT_ID
+export TF_VAR_operator_client_secret=$ARM_CLIENT_SECRET
+export TF_VAR_operator_tenant_id=$ARM_TENANT_ID
+
+echo "ARM_CLIENT_ID=$ARM_CLIENT_ID"
+echo "ARM_CLIENT_SECRET=$ARM_CLIENT_SECRET"
+echo "ARM_SUBSCRIPTION_ID=$ARM_SUBSCRIPTION_ID"
+echo "ARM_TENANT_ID=$ARM_TENANT_ID"
+echo "-----------"
+echo $TF_VAR_operator_subscription_id
+echo $TF_VAR_operator_client_id
+echo $TF_VAR_operator_client_secret
+echo $TF_VAR_operator_tenant_id
+
+echo "Terraform Bootstrap New Tenant"
+cd ../../terraform/providers/bootstrap_new_tenant
+terraform init
+terraform plan -var "namespace=$1" -out out.plan .
+terraform apply out.plan
+export REGISTRY_NAME=$(terraform output operations_container_registry_login_server)
+# export the container registry value for rhel.
+
+echo "Terraform Bootstrap"
+cd ../bootstrap
+terraform init
+terraform plan -out out.plan
+terraform apply out.plan
+
+echo "Building RHEL"
+cd ../../../ops/phase1
+[ ! -d "atat-rhel-image" ] && git clone git@github.com:dod-ccpo/atat-rhel-image.git
```

Comment:
As you know, there is the problem of accessing the private repo when `git` hasn't been set up for  the right user. What we did on Thursday was clone over HTTPS, which will prompt the user to input their username and password, avoiding the need to generate ssh keys or anything.

Due to 2FA simply using their password will fail, and the user will need to generate a [Personal Access Token](https://github.com/settings/tokens), which isn't hard to do, but we may want to prompt the user to do with an echo

```suggestion
if [ ! -d "atat-rhel-image" ]; then
	echo "Use a Personal Access Token instead of your password to auhenticeated with Github (https://github.com/settings/tokens)"
	git clone https://github.com/dod-ccpo/atat-rhel-image.git
fi
`````